public/app/features/explore/Wrapper.tsx

```diff
@@ -6,13 +6,32 @@ import { lastSavedUrl, resetExploreAction, richHistoryUpdatedAction } from './st
 import { getRichHistory } from '../../core/utils/richHistory';
 import { ExplorePaneContainer } from './ExplorePaneContainer';
 import { GrafanaRouteComponentProps } from 'app/core/navigation/types';
+import { NavModel } from '@grafana/data';
+import { Branding } from '../../core/components/Branding/Branding';
 
-interface WrapperProps extends GrafanaRouteComponentProps<{}, ExploreQueryParams> {
+// Libraries
+import { getNavModel, getTitleFromNavModel } from '../../core/selectors/navModel';
+
+// Types
+import { StoreState } from 'app/types';
```

Comment:
Please don't add import comments, that's something we're trying to remove. Thank you.

Reply:
Fixed :) https://github.com/grafana/grafana/pull/34651/commits/0e6990946b6134881f68061c97c19d295ff8976d#diff-06aad0593d46c656f9c92b1a6e517f30e9758c1f1eff827194d81389ee5c74f6R12